packages/ui-library/src/components/ui/icon/index.ts

```diff
@@ -8,15 +8,18 @@ import { ClassMapDirective } from 'lit-html/directives/class-map';
 import { until } from 'lit-html/directives/until.js';
 import { unsafeSVG } from 'lit-html/directives/unsafe-svg.js';
 import { TAG_NAME } from './renderFunction';
+import { ThemeType } from '../../../foundation/_tokens-generated/index.themes';
 
 @customElement(TAG_NAME)
 export class BlrIcon extends LitElement {
   static styles = [styleCustom];
 
   @property() icon: IconType = 'blr360Xs';
   @property() size: SizesType = 'md';
-
+  @property() arialabel?: string;
```

Comment:
we intentionally removed this one, because we figured out a way how to add html attributes which are not component properties the right way. so we will not need it as property anymore

Reply:
done

Reply:
So we also dont need it reperesentet anyomre in the documentation file as well?

Reply:
yes :)